### PR TITLE
Regenerate toke.c using updated flex

### PR DESCRIPTION
**Pull request by AtariDreams**
Use the current version of flex to generate toke.c

### CODE REVIEW

File: plugins/sudoers/toke.l [hunk 1]
```diff
@@ -1020,13 +1020,13 @@ read_dir_files(const char *dirpath, struct path_list ***pathsp, int verbose)
 	paths[count++] = pl;
     }
     closedir(dir);
-    if (count == 0) {
+    if (count <= 0) {
```
Comment (millert) [inline plugins/sudoers/toke.l:1023]: Now that it is size_t, count cannot be < 0 do this change should be dropped.

File: plugins/sudoers/toke.l [hunk 2]
```diff
@@ -1038,22 +1038,22 @@ bad:
...
 switch_dir(struct include_stack *stack, char *dirpath, int verbose)
 {
     struct path_list **paths = NULL;
-    int count, i;
+    size_t count, i;
     debug_decl(switch_dir, SUDOERS_DEBUG_PARSER);
 
     count = read_dir_files(dirpath, &paths, verbose);
-    if (count > 0) {
+    if (count) {
```
Comment (millert) [inline plugins/sudoers/toke.l:1056]: Another case where count cannot be < 0 so the original code is correct.

File: plugins/sudoers/toke.l [hunk 3]
```diff
@@ -1133,14 +1133,14 @@ strlcpy_expand_host(char *dst, const char *src, size_t size)
 		n = size ? size - 1 : 0;
 	    }
 	    dst += n;
-	    size -= n;
```
Comment (millert) [inline plugins/sudoers/toke.l:1136]: There is no need to change the order here.  I prefer it the way it currently is.

File: plugins/sudoers/toke.l [hunk 4]
```diff
@@ -1133,14 +1133,14 @@ strlcpy_expand_host(char *dst, const char *src, size_t size)
...
 	    src++;
+	    size -= n;
 	    continue;
 	}
 	if (size > 1) {
 	    *dst++ = ch;
-	    size--;
```
Comment (millert) [inline plugins/sudoers/toke.l:1142]: There is no need to change the order here.  I prefer it the way it currently is.

**Review by millert — CHANGES_REQUESTED** [review]

**Comment by AtariDreams** [comment]
@millert Done!

**Review by millert — APPROVED** [review]
Looks good to me

**Pull request merged**